docs/reference/transform/overview.asciidoc

```diff
@@ -62,3 +62,20 @@ image::images/pivot-preview.jpg["Example of a {transform} pivot in {kib}"]
 IMPORTANT: The {transform} leaves your source index intact. It
 creates a new index that is dedicated to the transformed data.
 
+
+[[transform-performance]]
+==== Performance considerations
+
+As mentioned earlier, {transforms} perform search aggregations on the source 
```

Comment:
I don't think you need this introductory phrase, since folks might jump to this section
```suggestion
{transforms} perform search aggregations on the source 
```